Type errors due to lack of types in bundled NPM package

**@datepicker-react/hooks or @datepicker-react/styled version**
@datepicker-react/styled@2.0.2

**Describe the bug**
Running typechecking (e.g. `tsc --noEmit`) with library inluded in a project codebase causes type errors:

```
node_modules/@datepicker-react/styled/lib/components/DateSingleInput/DateSingleInput.d.ts:4:41 - error TS2307: Cannot find module '@datepicker-react/hooks/src'.
```

Problem is caused by importing `@datepicker-react/hooks/src` module that is available in the source code but not available in the bundled NPM package.

--- a/packages/styled/src/components/DateSingleInput/DateSingleInput.tsx
+++ b/packages/styled/src/components/DateSingleInput/DateSingleInput.tsx
@@ -5,6 +5,8 @@ import {
   FormatFunction,
   getInputValue,
   OnDatesChangeProps,
+  FirstDayOfWeek,
+  MonthType,
 } from '@datepicker-react/hooks'
 import {dateSingleInputPhrases, DateSingleInputPhrases} from '../../phrases'
 import Box from '../Box'
@@ -13,7 +15,6 @@ import Datepicker from '../Datepicker'
 // eslint-disable-next-line import/no-unresolved
 import {DateSingleInputTheme} from '../../@types/theme'
 import useThemeProps from '../../hooks/useThemeProps'
-import {FirstDayOfWeek, MonthType} from '@datepicker-react/hooks/src'
 
 interface RtlProps {
   rtl: boolean

